docs/Zigbee.md

```diff
@@ -1,4 +1,5 @@
-!!! info "Zigbee2Tasmota serves as a gateway for devices connected to a Zigbee wireless network to bridge their communications over to Wi-Fi"
+<!-- Required extensions: pymdownx.betterem, pymdownx.tilde, pymdownx.emoji, pymdownx.tasklist, pymdownx.superfences -->
```

Comment:
why is that here?

Reply:
No idea, I think it might have been added by my editor tool :(